modules/programs/mcfly.nix

```
@@ -30,6 +48,8 @@ in {
       '';
     };
 
+    mcfly-fzf.enable = mkEnableOption "McFly fzf integration";
```

I think you could omit the `mcfly-` from the option name, it seems a bit redundant.
```suggestion
    fzf.enable = mkEnableOption "McFly fzf integration";
```

---

Yeah it does, just went ahead and added the change